ember-toucan-core/src/components/button.ts

```diff
@@ -36,9 +36,24 @@ const STYLES = {
 
 export interface ButtonSignature {
   Args: {
+    /**
+     * Sets the disabled attribute on the button.
```

Comment:
Nit: should we explicitly call out that setting `disabled` as an attribute is _not_ supported?

Reply:
Love it, thanks for the suggestion!